src/platform_impl/linux/wayland/seat/dnd/handlers.rs

```diff
@@ -0,0 +1,136 @@
+use std::{
+    io::{self, BufRead, BufReader},
+    path::PathBuf,
+};
```

Comment:
Could you use the style of the wayland module and group by module? Should apply everywhere.